hilti/toolchain/include/compiler/detail/optimizer/optimizer.h

```diff
@@ -58,19 +58,8 @@ class ASTState {
         return *_pinfo;
     }
 
-    /**
-     * Returns the control flow graph for the given block.
-     *
-     * The CFG is created on first request and cached for subsequent calls,
-     * until the block or its containing function/module is modified.
-     *
-     * @param block the block to get the CFG for, which must be part of a
-     * function or module
-     * @return the CFG (which will actually be the CFG for the outermost block
-     * containing the given block, i.e., the function or module body containing
-     * it)
-     */
-    CFG* cfg(statement::Block* block);
+    /** Returns the CFG cache in use. */
+    auto* cfgs() { return _cfgs; }
```

Comment:
Probably nit: `cfgs`/`_cfgs` does look like a great identifier to me. How about calling this e.g., `cfg_cache`? From a user's perspective they use this more as a cache to look up a CFG than a list of CFGs.